diff --git a/build.rs b/build.rs
index 27dfb57..5d944db 100644
--- a/build.rs
+++ b/build.rs
@@ -104,6 +104,7 @@ fn main() {
             "aarch64-unknown-linux-gnu" => "aarch64-linux-gnu",
             "x86_64-unknown-linux-musl" => "x86_64-linux-musl",
             "aarch64-unknown-linux-musl" => "aarch64-linux-musl",
+            "aarch64-linux-android" => "aarch64-linux-android",
             "x86_64-apple-darwin" => "x86_64-macos",
             "aarch64-apple-darwin" => "aarch64-macos",
             "x86_64-pc-windows-msvc" => "x86_64-windows-msvc",
